src/v2/components/FireListMapControl.jsx

```diff
@@ -0,0 +1,59 @@
+// -----------------------------------------------------------------------------
+// Copyright 2022 Open Climate Tech Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// -----------------------------------------------------------------------------
+
+import React, {useCallback, useMemo} from 'react'
+import ReactDOM from 'react-dom'
+
+import ButtonGroup from './ButtonGroup.jsx'
+import IconButton from './IconButton.jsx'
+
+import query from '../modules/query.mjs'
+
+/**
+ * @returns {React.Element}
+ */
+export default function FireListMapControl(props) {
+  const {container, isAuthenticated, map, onToggleAuthn} = props
+
+  const signIn = useCallback(() => {
+    onToggleAuthn('Sign in', () => onToggleAuthn(null, null, true))
+  }, [onToggleAuthn])
+
+  const signOut = useCallback(() => {
+    query.get('/api/logout').then(() => window.location.reload())
+  }, [])
+
+  const jsx = useMemo(() => {
+    if (container == null || map == null) {
+      return null
+    }
+
+    return 0,
+    <div className="c7e-map--control">
+      <ButtonGroup>
+        <IconButton icon='c7e-icon--zoom-in' label="Zoom in" title="Zoom in" onClick={() => map.zoomIn()}/>
+        <IconButton icon='c7e-icon--zoom-out' label="Zoom out" title="Zoom out" onClick={() => map.zoomOut()}/>
+      </ButtonGroup>
+
+      { isAuthenticated
+        ? <IconButton icon='c7e-icon--sign-out' label="Sign out" title="Sign out" onClick={signOut}/>
+        : <IconButton icon='c7e-icon--sign-in' label="Sign in" title="Sign in" onClick={signIn}/>
+      }
```

Comment:
I'm used to seeing login/logout at the top right.  Is your plan to move it later?

Reply:
Yeah, I’m not sure. One reason to have it top-left is to be closest to the vote buttons (which I am considering swapping positions with the pagination buttons). That way the sign-out is immediately next to the most likely thing to trigger a sign-in. But, you’re absolutely correct, people will look for sign-in/-out in the top-right. So maybe we should put it there along with the settings/preferences button, and put the zoom controls bottom-right (where Google Maps has them—in both cases no less).

<img width="600" alt="Google Maps 2022-08-25 17-59-02" src="https://user-images.githubusercontent.com/377413/186794248-39f945bd-7350-41b9-9afe-8aea62f3f1d1.png">